src/App.tsx

```diff
@@ -1,25 +1,30 @@
 import Home from "./components/pages/Home";
 import UserProfile from "./components/pages/Profile";
 import Navbar from "./components/Navbar";
+import { useState } from "react";
+import { TezosToolkit } from "@taquito/taquito";
 import React from "react";
 import { BrowserRouter, Switch, Route } from "react-router-dom";
-
+import { HoundInfo } from "./scripts/hound-genome";
+const temporaryHounds: HoundInfo[] = [];
 function App(): JSX.Element {
+  const [Tezos, setTezos] = useState<TezosToolkit>(new TezosToolkit("https://api.tez.ie/rpc/granadanet"));
+  const [hounds, setHounds] = useState(temporaryHounds);
```

Comment:
Would it be better to use a context for this instead? Since a horde of components may need access to this `Tezos` Singleton in the future. We can use a context with a setter. 

```ts
type TezosContext = {
   tezosToolkit: TezosToolKit | null,
   setTezosToolkit: (tezos: TezosToolKit) => void
};